modules/exploits/multi/local/vagrant_synced_folder_vagrantfile_breakout.rb

```diff
@@ -0,0 +1,121 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Msf::Post::File
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Vagrant Synced Folder Vagrantfile Breakout',
+        'Description' => %q{
+          This module exploits a default Vagrant synced folder (shared folder)
+          to append a Ruby payload to the Vagrant project Vagrantfile config file.
+
+          By default, unless a Vagrant project explicity disables shared folders,
+          Vagrant mounts the project directory on the host as a writable 'vagrant'
+          directory on the guest virtual machine. This directory includes the
+          project Vagrantfile configuration file.
+
+          Ruby code within the Vagrantfile is loaded and executed when a user
+          runs any vagrant command from the project directory on the host,
+          leading to execution of Ruby code on the host.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'HashiCorp', # Vagrant defaults
+          'bcoles' # Metasploit
+        ],
+        'DisclosureDate' => '2011-01-19', # Vagrant 0.7.0 release date - first mention of shared folders in CHANGELOG
+        'Platform' => %w[ruby],
+        'Arch' => ARCH_ALL,
+        'SessionTypes' => [ 'shell', 'powershell', 'meterpreter' ],
```

Comment:
```suggestion
        'SessionTypes' => [ 'shell', 'powershell', 'meterpreter' ],
        'Stance'         => Msf::Exploit::Stance::Passive,
```
Since we're waiting on a user to launch, this feels like a passive stance.

Reply:
Added